python/cudf/cudf/core/indexed_frame.py

```diff
@@ -1639,6 +1639,12 @@ def hash_values(self, method="murmur3"):
             * murmur3: MurmurHash3 hash function.
             * md5: MD5 hash function.
 
+        seed : int, optional
+            Seed value to use for the hash function.
+            Note - This only has effect for the following supported
+            hash functions:
+            * murmur3: MurmurHash3 hash function.
```

Comment:
If the seed has no effect, I think maybe we should warn, at least since we have some flexibility here given this has no equivalent pandas API.